wptrainingteam/devblog-dataviews-plugin
# Fix dataviews bundle

Fixes https://github.com/wptrainingteam/devblog-dataviews-plugin/issues/2

## What

This PR uses the approach implemented at https://github.com/WordPress/gutenberg/pull/66825 to bundle every dependency required by DataViews.

## Why

To prevent conflicts from having different versions of the underlying libraries (e.g.: `@wordpress/components`).

## How

- Use the `next` versions of `@wordpress/scripts` and `@wordpress/dataviews`. The patch hasn't been released to the stable version yet.
  - `npm install --save @wordpress/dataviews@next`, which today results in this version https://github.com/wptrainingteam/devblog-dataviews-plugin/pull/3/commits/72b3c615ab5c92e68dd4299a82b224ff334d7b56
  - `npm install --save-dev @wordpress/scripts@next`, which today results in this version https://github.com/wptrainingteam/devblog-dataviews-plugin/pull/3/commits/e9d7fcb61c8a5989035e19258cdc3c321de704a5
- Import dataviews from `@wordpress/dataviews/wp`. See https://github.com/wptrainingteam/devblog-dataviews-plugin/pull/3/commits/18fcb6e8a52774fe546e3ad0dba3e75cfc094e11

## How to test

Use WordPress 6.6 or WordPress 6.7, with and without the Gutenberg plugin active. Make sure the plugin still works.
